rules/library/xcconfig.bzl

```diff
@@ -112,12 +145,7 @@ def _add_copts_from_option(xcspec, option, value, value_escaper, copts, linkopts
         new = [command_line_prefix_flag + "$(value)"]
 
     if new == None:
-        fail("{name}: unable to extract value for {value} in {xcspec}\n\t{option}".format(
```

Comment:
Looks like you're explicitly expecting this now with `Command LineFlag` setting `new = None`. Was this catching anything else that will now no longer be caught?

Reply:
Not that I know of -- now that we support conditions, there are some options that are _only_ used as options